peopleatschool.rb

```diff
@@ -0,0 +1,35 @@
+class PeopleAtSchool
+ attr_reader :age, :salary, :phase, :target_raise
+ attr_accessor :name
+
+   def initialize(options={})
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+    @target_raise = 80
+    @phase = 3
+end
+
+ def offer_high_five
+    "High five!"
+  end
+
+  def teach_stuff
+    response = ""
```

Comment:
The teach_stuff method is not the same for both teachers.  Not a candidate for putting in a super class.